recipe/meta.yaml

```
@@ -1,15 +1,7 @@
-{% set name = "or-tools" %}
-{% set package_name = "ortools" %}
 {% set version = "9.2" %}
-{% set home = "https://developers.google.com/optimization" %}
-{% set license = "Apache-2.0" %}
-{% set license_family = "Apache" %}
-{% set license_file = "LICENSE-2.0.txt" %}
-{% set dev_url = "https://github.com/google/or-tools" %}
-
 
 package:
-  name: {{ name|lower }}-package
+  name: or-tools
```

```suggestion
  name: or-tools-package
```
No it is useful to have this be a different names